frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -238,19 +240,57 @@ const ProgressStatDescription = styled.p`
   }
 `;
 
+const DayLabel = styled.p<DayLabelType>`
+  font-weight: ${(props) => (props.isToday ? 700 : 400)};
```

Comment:
Can we destructure this please? Also did not know you could do styledcomponents with the type there, I think that looks super nice!!

Reply:
An example of destructuring in a styled component can be found in the YesNoSelection component